docs/conf.py

```diff
@@ -174,6 +174,7 @@
     "issues": "https://github.com/Pycord-Development/pycord/issues",
     "discussions": "https://github.com/Pycord-Development/pycord/discussions",
     "examples": f"https://github.com/Pycord-Development/pycord/tree/{branch}/examples",
+    "guide": "https://guide.pycord.dev/"
```

Comment:
trailing commas make sense for VCS so git diffs are only one line long, like how it is right now